client/js/admin.js

```diff
@@ -38,7 +39,86 @@ $(function() {
 			return false;
 		}
 	});
-})
+
+	$("#uploadTrigger").click(function(e) {
+		var form = $("#fileUpload")
+		form.off("change")
+		form.on("change", () => {
+			var XHR = new XMLHttpRequest()
+			XHR.open("POST", "/static/")
+			var fData = new FormData(form[0].form)
+			XHR.onload = function() {
+				var res = $(".upload-name.upload-item")
+				res.removeClass("disabled")
+				var out = JSON.parse(XHR.response)
+				res.text(out.path)
+			}
+			XHR.send(fData)
+		})
+		form.click()
+	})
+
+	$("#filePageTrigger").click(function(e) {
+		var form = $("#fileUpload")
+		form.off("change")
+		form.on("change", () => {
+			var XHR = new XMLHttpRequest()
+			XHR.open("POST", "/static/")
+			var fData = new FormData(form[0].form)
+			XHR.onload = function() {
+				var dummy = $(".file-item:first-child")[0].outerHTML
```

Comment:
Oh, this element is a dummy placed in there. Its a way to copy the server side template without so I can do a client side animation. Only necessary if you want to reload the page after file upload. 

Bear in mind, this is close to how templating engines work.